apps/coscrad-frontend-e2e/src/e2e/resources/photographs/queries/photograph.detail.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,151 @@
+import { AggregateType, LanguageCode, MIMEType } from '@coscrad/api-interfaces';
```

Comment:
Happy to see us keeping up with Cypress test coverage! Thanks for being diligent.